### PR TITLE
corrected for submission

### PR DESCRIPTION
**Pull request by gallg**
1. fixed affiliation in myst.yaml
2. example.ipynb has been rerun
3. package version and package requirements fixed

### CODE REVIEW

File: manuscript/myst.yml [hunk 1]
```diff
@@ -8,40 +8,40 @@ project:
   authors:
     - name: Giuseppe Gallitto
       affiliations:
-        - Center for Translational Neuro- and Behavioral Sciences (C-TNBS), University Medicine Essen, Germany
-        - Department of Neurology, University Medicine Essen, Germany
+        - Center for Translational Neuro- and Behavioral Sciences (C-TNBS), University Hospital Essen, Germany
```
Comment (spisakt) [inline manuscript/myst.yml]: University Medicine Essen is the correct one.

**Review by spisakt — CHANGES_REQUESTED** [review]
University Medicine Essen is the correct affiliation, actually.

File: manuscript/myst.yml (hunk 1 above)
Comment (gallg) [reply manuscript/myst.yml]: done

**Review by spisakt — APPROVED** [review]
thank you

**Pull request merged**